docs/cheaha/software/software.md

```diff
@@ -52,6 +52,82 @@ For more information on usage with examples, see [Anaconda Environments](../../w
 
 Please see our [CUDA and cuDNN section of the GPU page](../slurm/gpu.md#cuda-and-cudnn-modules).
 
+## Cell Ranger
+
+Cell Ranger is a proprietary software developed by [10x Genomics](https://www.10xgenomics.com/support/software/cell-ranger/latest).
+
+### Licensing
+
+(i) Usage of Cell Ranger is governed by the terms of their [End User License Agreement (EULA)](https://www.10xgenomics.com/legal/end-user-software-license-agreement).
+
+(ii) Cell Ranger is strictly licensed for use with data generated on 10x Genomics instruments and may not be used with data from other platforms.
+
+### Installation
+
+ Cell Ranger can be installed within a researcher’s individual user account on Cheaha. Installation instructions are available on the official [10x Genomics website](https://www.10xgenomics.com/support/software/cell-ranger/latest/tutorials/cr-tutorial-in#download). You will have to use a compute node in Cheaha to install and run the software. For more information refer to [Login Vs Compute Nodes](../../cheaha/getting_started.md#login-vs-compute-nodes). Following are the steps to install Cell Ranger in Cheaha.
```

Comment:
Start a new paragraph with the last sentence, changing it to "Following are steps to install Cell Ranger in Cheaha, based on the instructions provided in the official site, linked above, as of 2025-05-01. Actual instruction steps may change over time."